mne/channels/tests/test_montage.py

```diff
@@ -103,7 +103,7 @@ def test_montage():
             assert_array_less(distance_from_centroid, 0.2)
             assert_array_less(0.01, distance_from_centroid)
 
-    # test transform
+    # test transform with hpts [ fiducials == ('2', '1', '3') ]
```

Comment:
hehe, I meant the pg 231 in the pdf using absolute numbering versus the labeled page number